server/models/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Sequelize } from 'sequelize-typescript';
+import { Chapter } from './chapter';
+
+export const initSequelize = () =>
+  new Sequelize({
+    dialect: 'postgres',
+    username: 'postgres',
+    password: 'password',
+    database: 'chapter',
+    host: 'db',
+    port: 5432,
+    models: [Chapter],
```

Comment:
instead of defining models like this, we can dynamically load and define models. Also, what happens if we have associations in a model?